pretyping/coercion.ml

```diff
@@ -35,6 +35,15 @@ let use_typeclasses_for_conversion = ref true
 let _ =
   Goptions.(declare_bool_option
     { optdepr  = false;
+      optname  = "use typeclass resolution during conversion";
+      optkey   = ["Typeclasses"; "Resolution"; "For"; "Conversion"];
+      optread  = (fun () -> !use_typeclasses_for_conversion);
+      optwrite = (fun b -> use_typeclasses_for_conversion := b) }
+    )
+
+let _ =
+  Goptions.(declare_bool_option
+    { optdepr  = true; (* alias of see above, remove in 8.8 *)
```

Comment:
I think the other one (`Typeclasses Resolution`) should be deprecated, and the documentation should be updated.

Reply:
The other one didn't exist before this PR IIRC. So what you want in fact is that the documentation be updated according to the existing options, instead of the option being renamed to match the documentation.